TheDreamProject/vision_app/views.py

```diff
@@ -70,7 +70,13 @@ def get(self, request):
         
         return render(request, 'addgoaltask.html',{
                 '   Goal_Form' : Goal_Form,
-            })       
+            })   
+
+    def post(self,request):
+        '''POST the data in the form submitted by the user, creating a new task in the todo list'''
+        Goal_Form = GoalForm(request.POST)
+        Goal_Form.save()
```

Comment:
```suggestion
        if Goal_Form.is_valid():
            Goal_Form.save()
```

Reply:
I'm still a little unclear on how this works more generally -- it seems strange that a `Goal_Form` is being submitted when I had just clicked submit to create a _vision_? Any idea why this is?

Reply:
When the user creates a vision using the Vision_Form it suppose to create that object in the database Vision model. After the data has been submitted the vision_name is rendered to the addgoaltask.html. 

Everything looks good for as models, forms and django forms. I might be missing something.